management-app/src/main/java/com/nearsoft/fgaribay/mgmt/BrokerConnector.java

```diff
@@ -0,0 +1,54 @@
+package com.nearsoft.fgaribay.mgmt;
+
+import com.nearsoft.fgaribay.mgmt.model.Product;
+import com.nearsoft.fgaribay.mgmt.model.ProductRequest;
+import com.nearsoft.fgaribay.mgmt.model.ProductResponse;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class BrokerConnector {
+    private final RabbitTemplate rabbitTemplate;
+    @Value("product.create.queue.name")
+    private String productCreationQueueName;
+    @Value("product.list.queue.name")
+    private String productListQueueName;
+
+    @Autowired
```

Comment:
Since you're using constructor injection the `@Autowired` annotation is not needed. In general, we prefer to use constructor injection instead of field injection, so check your Spring beans and make changes according to this.